projects/dsp-ui/src/lib/viewer/views/property-view/property-view.component.spec.ts

```diff
@@ -1,6 +1,13 @@
 import { Component, Input, OnInit, ViewChild } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { IHasProperty, MockResource, PropertyDefinition, ReadResource, ReadValue, ResourcePropertyDefinition, SystemPropertyDefinition } from '@knora/api';
+import {
+    ApiResponseError,
+    IHasPropertyWithPropertyDefinition,
+    MockResource,
+    ReadResource,
+    ReadValue,
+    SystemPropertyDefinition
+} from '@knora/api';
 import { PropertyInfoValues } from '../resource-view/resource-view.component';
 import { PropertyViewComponent } from './property-view.component';
 
```

Comment:
a comment referring to `knora-ui-ng-lib/projects/dsp-ui/src/lib/viewer/views/property-view/property-view.component.ts`:

```ts
  /**
   * Array of property object with ontology class prop def, list of properties and corresponding values
   *
   * @param (propArray)
   */
  @Input() propArray: PropertyInfoValues;

  /**
   * Array of system property object with list of system properties
   *
   * @param (propArray)
   */
  @Input() systemPropArray?: PropertyDefinition;
```

Shouldn't these be arrays `PropertyInfoValues[]` and `PropertyDefinition[]`? Also I think `systemPropArray` needn't be optional, the array could just be empty.

Reply:
resolve in commit [b452f0e](https://github.com/dasch-swiss/knora-ui-ng-lib/pull/88/commits/b452f0e6304f8b8665c15d593c51a6b5398a18c1)